sounds/m.md

```diff
@@ -51,20 +51,24 @@ Right side M is made with the combination `-PL`. This _does_ conflict with words
 | HEP              | help        | HEPL is "hem"                                                                                                                                               |
 | SKHRAPL or TP-BG | {!}         | The exclamation mark is written with "sklam". Alternatively, TP-BG can be used.                                                                             |
 | KW-PL            | {?}         | Alternative stroke for question mark. Some users prefer `TP-BG` and `KW-PL` as they can represent the intonation speakers use during a question or exclamation. |
+|
+| PHAUL            | maul        |
```

Comment:
Thanks for the change. I think the PHAUL == maul note way up top needs to be changed to "mall", and I'd like to leave "maul" out of the brief list for now.